arangod/Cluster/AgencyCallback.cpp

```diff
@@ -89,35 +89,18 @@ void AgencyCallback::refetchAndUpdate(bool needToAcquireMutex,
   LOG_TOPIC("a6344", TRACE, Logger::CLUSTER)
       << "Refetching and update for " << AgencyCommHelper::path(key);
 
-  if (_local) {
-    auto& _cache = _server.getFeature<ClusterFeature>().agencyCache();
-    std::tie(builder, idx) =
-        _cache.read(std::vector<std::string>{AgencyCommHelper::path(key)});
-    result = builder->slice();
-    if (!result.isArray()) {
-      if (!_server.isStopping()) {
-        // only log errors if we are not already shutting down...
-        // in case of shutdown this error is somewhat expected
-        LOG_TOPIC("ec320", ERR, arangodb::Logger::CLUSTER)
-            << "Callback to get agency cache was not successful: "
-            << result.toJson();
-      }
-      return;
+  std::tie(builder, idx) =
```

Comment:
There is no need anymore to use `std::tie(builder, idx)` here to assign data to variables declared above.
Instead we could use `auto [builder, idx] = ...` here and remove the separate declarations above.
We then also can remove the now-unused variable `tmp` above.